pkg/virt-controller/watch/vmi/vmi_test.go

```diff
@@ -26,6 +26,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/google/uuid"
```

Comment:
nit: move to import block below

Reply:
Can you clarify which import block you're referring to?
I see only 1 import block, which this import is in already.
The import statements just below this line have a leading period. Are you suggesting that we should declare `uuid`'s identifiers in the test file itself ?

Reply:
I meant move it to L36, but ideally automation should do that for us.

Reply:
Thank you for clarifying. Fixed it.